cmd/zqd/winexec/exec_windows.go

```diff
@@ -0,0 +1,68 @@
+package winexec
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"unsafe"
+
+	"github.com/alexbrainman/ps"
+	"github.com/alexbrainman/ps/winapi"
+)
+
+// ensureSpawnedProcessTermination ensures that when this Go process terminates, the
```

Comment:
fyi, this is all taken from the launcher utility already in use on windows:
https://github.com/brimsec/zeek/blob/master/brim/windows/launcher/zeek-launcher.go#L72